### DISCUSSION TITLE
GSoC interest for Optimizing backend kernels in PyElastica

### DISCUSSION THREAD

**Discussion by octonawish-akcodes** [GazzolaLab/PyElastica#338] [Q&A]
Hi team, I am interested in this following project idea for this GSoC term:

Optimizing backend kernels in PyElastica

How can I get started? @armantekinalp @bhosale2

**Comment by armantekinalp** [comment]
You can start by checking the linear algebra kernels in [_linalg.py](https://github.com/GazzolaLab/PyElastica/blob/master/elastica/_linalg.py). These are the some of the kernels we are planning to improve.

Also do a literature search on possible accelerators i.e. jax etc and think about how can use them to improve the speed of kernels.
